packages/specs/schema/src/decorators/common/default.spec.ts

```
@@ -22,4 +22,24 @@ describe("@Default", () => {
       type: "object"
     });
   });
+  it("should declare prop (Date.now)", () => {
+    // WHEN
+    class Model {
+      @Default(Date.now)
+      num: Date = new Date();
+
+      constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)